installer/I05_emba_docker_image_dl.sh

```diff
@@ -48,7 +48,13 @@ I05_emba_docker_image_dl() {
         if command -v docker > /dev/null ; then
           export DOCKER_CLI_EXPERIMENTAL=enabled
           echo -e "$ORANGE""EMBA docker image will be downloaded.""$NC"
-          docker pull embeddedanalyzer/emba
+          if [ -z "${CONTAINER+isset}" ]; then
+            docker pull embeddedanalyzer/emba
```

Comment:
should this whole if clause be switched around?
The default case should be to use `docker pull embeddedanalyzer/emba`
In case CONTAINER is set it should be used

I think something like the following could work:
```
if [[ -v CONTAINER ]]; then
  # your else condition
else
  docker pull embeddedanalyzer/emba
fi
```
What do you think?

Reply:
Yes, but if I understood it correctly, the problem is not the if statement. The problem is line 29 when the size of docker image gets calculated.

Reply:
> 

Yes, I will do it so.